Add Arbiter (#210)
SteamDatabase/FileDetectionRuleSets: descriptions/AntiCheat.Arbiter.md

> Halo Infinite is shipped with its own proprietary anti-cheat system which we call [**Arbiter**](https://www.halowaypoint.com/news/closer-look-halo-infinites-anti-cheat-strategy).

Reviewer: i would prefer something like that
```suggestion
[**Arbiter**](https://www.halowaypoint.com/news/closer-look-halo-infinites-anti-cheat-strategy) is the anti-cheat system used by Halo Infinite.
```

Author: done
